packages/dashboard/pages/create.tsx

```diff
@@ -56,6 +56,7 @@ const Resolver: NextPage = () => {
     const didResolution = await response.json();
     wallet.add(didResolution);
     localStorage.setItem('sidetree.wallet', JSON.stringify(wallet));
+    setWallet(wallet);
```

Comment:
change unrelated?

Reply:
Not related but it should show the did it created after the API call